docroot/themes/custom/uids_base/templates/block/block--inline-block--uiowa-image.html.twig

```diff
@@ -0,0 +1,51 @@
+{#
+/**
+ * @file
+ * Theme override to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ */
+#}
+{%
+  set classes = [
+  'block',
+  'block-' ~ configuration.provider|clean_class,
+  'block-' ~ plugin_id|clean_class,
+]
+%}
+{% set content_rendered = content|render %}
+{% if content_rendered is not empty %}
+  <div{{ attributes.addClass(classes) }}>
+    {{ title_prefix }}
+    {% if label %}
+      <h2{{ title_attributes }}>{{ label }}</h2>
+    {% endif %}
+    {{ title_suffix }}
+    {% block content %}
+      {% if content.field_uiowa_image_link[0]['#url'] is not empty %}
+        <a href="{{ content.field_uiowa_image_link[0]['#url']|render }}" aria-label="{{ content.field_uiowa_image_link[0]['#title']|render }}">{{ content.field_uiowa_image_image }}</a>
+      {% else %}
+        {{ content.field_uiowa_image_image }}
+      {% endif %}
+    {% endblock %}
```

Comment:
This is the `{% endblock %}` that should be removed per previous comment.

Reply:
This has been removed.